### PR TITLE
plasma5: 5.21.1 -> 5.21.3

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Plasma bugfix release. Works here with the various plasma components needed to be my daily driver.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Cc: @samueldr @ttuegel @CertainLach

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117921` at 318a4f02 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>92 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.bluedevil</li>
    <li>libsForQt512.breeze-gtk</li>
    <li>libsForQt512.breeze-qt5</li>
    <li>libsForQt512.discover</li>
    <li>libsForQt512.kactivitymanagerd</li>
    <li>libsForQt512.kde-cli-tools</li>
    <li>libsForQt512.kde-gtk-config</li>
    <li>libsForQt512.kde2-decoration</li>
    <li>libsForQt512.kdecoration</li>
    <li>libsForQt512.kdenlive</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/dce165b8e42381c5d596ed6800079d1f">libsForQt514.libksysguard</a></li>
    <li><a href="https://gist.github.com/7df6929034e050e160be2f15aa6b5e2b">libsForQt514.plasma-vault</a></li>
  </ul>
</details>
<details>
  <summary>28 packages skipped due to time constraints:</summary>
  <ul>
    <li>digikam</li>
    <li>kdev-php</li>
    <li>kdev-python</li>
    <li>kdevelop-unwrapped</li>
    <li>libsForQt5.kde-cli-tools (libsForQt515.kde-cli-tools ,plasma5Packages.kde-cli-tools)</li>
    <li>libsForQt5.kdenlive (libsForQt515.kdenlive ,plasma5Packages.kdenlive)</li>
    <li>libsForQt5.kdeplasma-addons (libsForQt515.kdeplasma-addons ,plasma5Packages.kdeplasma-addons)</li>
    <li>libsForQt5.khotkeys (libsForQt515.khotkeys ,plasma5Packages.khotkeys)</li>
    <li>libsForQt5.kinfocenter (libsForQt515.kinfocenter ,plasma5Packages.kinfocenter)</li>
    <li>libsForQt5.kmenuedit (libsForQt515.kmenuedit ,plasma5Packages.kmenuedit)</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>38 packages built successfully:</summary>
  <ul>
    <li>libsForQt5.bluedevil (libsForQt515.bluedevil ,plasma5Packages.bluedevil)</li>
    <li>libsForQt5.breeze-grub (libsForQt515.breeze-grub ,plasma5Packages.breeze-grub)</li>
    <li>libsForQt5.breeze-gtk (libsForQt515.breeze-gtk ,plasma5Packages.breeze-gtk)</li>
    <li>libsForQt5.breeze-plymouth (libsForQt515.breeze-plymouth ,plasma5Packages.breeze-plymouth)</li>
    <li>libsForQt5.breeze-qt5 (libsForQt515.breeze-qt5 ,plasma5Packages.breeze-qt5)</li>
    <li>libsForQt5.discover (libsForQt515.discover ,plasma5Packages.discover)</li>
    <li>libsForQt5.kactivitymanagerd (libsForQt515.kactivitymanagerd ,plasma5Packages.kactivitymanagerd)</li>
    <li>libsForQt5.kde-gtk-config (libsForQt515.kde-gtk-config ,plasma5Packages.kde-gtk-config)</li>
    <li>libsForQt5.kde2-decoration (libsForQt515.kde2-decoration ,plasma5Packages.kde2-decoration)</li>
    <li>libsForQt5.kdecoration (libsForQt515.kdecoration ,plasma5Packages.kdecoration)</li>
    <li>libsForQt5.kgamma5 (libsForQt515.kgamma5 ,plasma5Packages.kgamma5)</li>
    <li>libsForQt5.kscreen (libsForQt515.kscreen ,plasma5Packages.kscreen)</li>
    <li>libsForQt5.kscreenlocker (libsForQt515.kscreenlocker ,plasma5Packages.kscreenlocker)</li>
    <li>libsForQt5.ksshaskpass (libsForQt515.ksshaskpass ,plasma5Packages.ksshaskpass)</li>
    <li>libsForQt5.kwallet-pam (libsForQt515.kwallet-pam ,plasma5Packages.kwallet-pam)</li>
    <li>libsForQt5.kwayland-integration (libsForQt515.kwayland-integration ,plasma5Packages.kwayland-integration)</li>
    <li>libsForQt5.kwayland-server (libsForQt515.kwayland-server ,plasma5Packages.kwayland-server)</li>
    <li>libsForQt5.kwin (libsForQt515.kwin ,plasma5Packages.kwin)</li>
    <li>libsForQt5.kwrited (libsForQt515.kwrited ,plasma5Packages.kwrited)</li>
    <li>libsForQt5.libkscreen (libsForQt515.libkscreen ,plasma5Packages.libkscreen)</li>
    <li>libsForQt5.milou (libsForQt515.milou ,plasma5Packages.milou)</li>
    <li>libsForQt5.plasma-integration (libsForQt515.plasma-integration ,plasma5Packages.plasma-integration)</li>
    <li>libsForQt5.plasma-pa (libsForQt515.plasma-pa ,plasma5Packages.plasma-pa)</li>
    <li>libsForQt5.plasma-thunderbolt (libsForQt515.plasma-thunderbolt ,plasma5Packages.plasma-thunderbolt)</li>
    <li>libsForQt5.plasma-workspace-wallpapers (libsForQt515.plasma-workspace-wallpapers ,plasma5Packages.plasma-workspace-wallpapers)</li>
    <li>libsForQt5.polkit-kde-agent (libsForQt515.polkit-kde-agent ,plasma5Packages.polkit-kde-agent)</li>
    <li>libsForQt5.sddm-kcm (libsForQt515.sddm-kcm ,plasma5Packages.sddm-kcm)</li>
    <li>libsForQt5.spectacle (libsForQt515.spectacle ,plasma5Packages.spectacle)</li>
    <li>libsForQt5.xdg-desktop-portal-kde (libsForQt515.xdg-desktop-portal-kde ,plasma5Packages.xdg-desktop-portal-kde)</li>
    <li>libsForQt512.breeze-grub</li>
    <li>libsForQt512.breeze-plymouth</li>
    <li>libsForQt512.plasma-workspace-wallpapers</li>
    <li>libsForQt514.breeze-grub</li>
    <li>libsForQt514.breeze-plymouth</li>
    <li>libsForQt514.kwallet-pam</li>
    <li>libsForQt514.plasma-thunderbolt</li>
    <li>libsForQt514.plasma-workspace-wallpapers</li>
    <li>plasma-applet-volumewin7mixer</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement.

File: pkgs/desktops/plasma-5/srcs.nix [hunk 1]
```diff
@@ -1,422 +1,422 @@
 # DO NOT EDIT! This file is generated automatically.
-# Command: ./maintainers/scripts/fetch-kde-qt.sh pkgs/desktops/plasma-5/
+# Command: ./maintainers/scripts/fetch-kde-qt.sh pkgs/desktops/plasma-5
```
Comment (SuperSandro2000) [inline pkgs/desktops/plasma-5/srcs.nix:2]: Is that supposed to change?

Comment (peterhoeg) [reply pkgs/desktops/plasma-5/srcs.nix:2]: I didn't touch the script and `srcs.nix` is 100% generated so I'll go with yes.

Comment (SuperSandro2000) [reply pkgs/desktops/plasma-5/srcs.nix:2]: Maybe it changes depending on the directory you supply.

**Comment by ttuegel** [comment]
I was already preparing to update all the KDE components in `staging` with #117947 due to the mass-rebuild-y nature of that update, but I guess we can merge your Plasma update directly into master because it doesn't touch Darwin. :+1:

File: pkgs/desktops/plasma-5/srcs.nix (hunk 1 above)
Comment (ttuegel) [reply pkgs/desktops/plasma-5/srcs.nix:2]: It does; the script doesn't really do any normalization of the directory name.

**Review by ttuegel — APPROVED** [review]

**Comment by ttuegel** [comment]
Thanks!

**Pull request merged**